docs/src/pages/components/dialogs/DialogWithPortalsInside.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Button, Dialog, TextField, Popper } from '@material-ui/core';
+
+function App() {
```

Comment:
If we want a new demo, we would probably want to simplify it. However, considering the <kbd>Tab</kbd> logic isn't correct (we can't tab in the logical order), I'm not sure we want a new demo.

Reply:
I wasn't planning of adding the demo, simply because I don't think this is something that we need to advertise (it's just a bug fix)... It may be misleading of what we want to show with the example.. But for testing it's good so I added it for easier review

Reply:
Sounds great :)

Reply:
Removing it